From e6ffc044f1e78ab9a75a18c2072df492a57d33e9 Mon Sep 17 00:00:00 2001
From: Michal Ambroz <723625+xambroz@users.noreply.github.com>
Date: Sun, 17 Dec 2023 13:57:34 +0100
Subject: [PATCH] remove unversioned shabeng

python shabeng not needed to this file and it was not even conclusive to require pytho2 or python3.
---
 ldapdomaindump/__main__.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldapdomaindump/__main__.py b/ldapdomaindump/__main__.py
index c0bb795..dfaba8f 100644
--- a/ldapdomaindump/__main__.py
+++ b/ldapdomaindump/__main__.py
@@ -1,3 +1,2 @@
-#!/usr/bin/env python
 import ldapdomaindump
 ldapdomaindump.main()
